freud/locality.pyx

```diff
@@ -1129,8 +1129,10 @@ cdef class Voronoi(_Compute):
                 :class:`freud.locality.NeighborQuery.from_system`.
         """
         cdef NeighborQuery nq = NeighborQuery.from_system(system)
-        self.thisptr.compute(nq.get_ptr())
         self._box = nq.box
+        radii = self.radii if radii is not None else np.zeros(nq.positions.shape[0])
+        cdef np.float32_t[::1] l_radii = radii
```

Comment:
`double` maps to `float64_t`, but you should use `double` directly instead of the NumPy type.
```suggestion
        cdef double[::1] l_radii = radii
```